code/modules/mob/mob.dm

```diff
@@ -362,7 +362,7 @@
 
 		add_logs(src, M, "grabbed", addition="passive grab")
 		if(!supress_message)
-			visible_message("<span class='warning'>[src] has grabbed [M] passively!</span>")
+			visible_message("<span class='warning'>[src] has grabbed [M][(zone_selected == "l_arm" || zone_selected == "r_arm")? " by their hands":" passively"]!</span>")
```

Comment:
this addition is a fucking crime

Reply:
1000 points gulag

Reply:
Speaking of points, the bot didn't give you a feature tag - this would have been free!